lib/draper/collection_decorator.rb

```diff
@@ -72,6 +71,17 @@ def replace(other)
       self
     end
 
+    # Proxies missing query methods to the source class if the strategy allows.
+    def method_missing(method, *args, &block)
+      return super unless strategy.allowed? method
```

Comment:
I'm new here, so please forgive my stupid question 🙏

Why do we need any _strategy_ in the first place? Isn't `object.respond_to?` enough?

Reply:
I see: it's for methods returning decoratable collections. It's not obvious for everyone, though (see #867).